rules/ssh.go

```diff
@@ -0,0 +1,47 @@
+// (c) Copyright 2016 Hewlett Packard Enterprise Development LP
```

Comment:
I am no longer am associated with HP and since this contribution is from you I don't think this header needs to be here. 

I probably need to create a LICENSE, AUTHORS, and COPYRIGHT files to better track contributions and copyright information.